src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1211,115 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-registry
+helps['acr connected-registry'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
+"""
+
+helps['acr connected-registry create'] = """
+type: command
+short-summary: Create a connected registry for an Azure Container Registry.
+examples:
+  - name: Create a connected registry in registry mode with access to repos app/hello-world and service/mycomponent. It'll create a sync token and scope-map with the right repo permissions.
+    text: |
+        az acr connected-registry create --registry mycloudregistry --name myconnectedregistry
+            --repository "app/hello-world service/mycomponent"
+  - name: Create a mirror connected registry with only read permissions and pass the sync token
```

Comment:
as a child to another connected registry

Reply:
Not sure, if we want to provide this option now as it is blocked on RP?/cc @jaysterp